### PR TITLE
Add RISC-V icon

### LINKED ISSUE #8777: Add RISC-V logo
### Brand Name

RISC-V

### Website

https://riscv.org/

### Popularity Metric

The Similarweb rank is 406,735. See https://www.similarweb.com/website/riscv.org

### Official Resources for Icon and Color

SVG: https://riscv.org/risc-v-logo/
Color: `#FFC72C`
Logo and use guidelines: https://riscv.org/about/risc-v-branding-guidelines/

### Additional Comments

RISC-V is an open standard Instruction Set Architecture(ISA). More than 3,100 RISC-V members across 70 countries contribute and collaborate to define RISC-V open specifications as well as convene and govern related technical, industry, domain, and special interest groups. Content taken from their own website: https://riscv.org/about/

It is the fifth version of RISC which has thrived since 1980s along proprietary ISAs like ARM, x86 and AMD64. Almost everything except desktop class CPUs are RISC based and even the desktop ones make use of RISC but that is out of scope here. RISC based CPUs go inside consumer electronics, automobiles and almost everything that has some controller inside it. 

### Contributing

Yes

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
riscv

**Issue:** closes #8777 

**Similarweb rank:** 406,735. See https://www.similarweb.com/website/riscv.org

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Hex Value picked from logo: `#FFC72C`
Logo provided as an EPS by the brand itself. Isolated logo and converted to monochromatic SVG.

### CODE REVIEW

**Comment by PratyushJayachandran** [comment]
Running into this error on local even on develop
```
Checking formatting...
[warn] scripts\add-icon-data.js
[warn] Code style issues found in the above file. Forgot to run Prettier?
ERROR: "jslint" exited with 1.
```

But checks seem to have passed on GitHub

**Comment by PratyushJayachandran** [comment]
I am not able to see the additional information I added about RISC-V. I don't know if the maintainers can see it. If needed, please leave a comment and I will be happy to add more details.
Basically RISC-V is an open-source Instruction Set Architecture (ISA) that has existed alongside proprietary ISAs like x86, ARM and AMD64. Find more info here: https://riscv.org/about/

File: _data/simple-icons.json [hunk 1]
```diff
@@ -10065,6 +10065,18 @@
...
+            "hex": "FFC72C",
+            "aliases": {
+                "aka": [
+                    "RISC",
+                    "RISC Five"
+                ]
+            },
+            "source": "https://riscv.org/risc-v-logo/",
+            "guidelines": "https://riscv.org/about/risc-v-branding-guidelines/"
+        },
```
Comment (LitoMore) [inline _data/simple-icons.json:10079]:
```suggestion
            "title": "RISC-V",
            "hex": "283272",
            "source": "https://riscv.org/risc-v-logo/",
            "guidelines": "https://riscv.org/about/risc-v-branding-guidelines/"
        },
```

Those aliases are unnecessary.

And the color should be `#283272`. They have an example of monochrome image.

image

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (PratyushJayachandran) [reply _data/simple-icons.json]: riscv

Updated and pushed the commit

**Review by LitoMore — CHANGES_REQUESTED** [review]
image

I found some differences. Could you point out which asset are you using as reference?

I used the `Tall_2.eps` as reference.

image

**Comment by PratyushJayachandran** [comment]
> I found some differences. Could you point out which asset are you using as reference?
> 

I used Standard-Black_2.eps and there seems to be some difference from Tall_2.eps
image

Which one should I use?

**Comment by PratyushJayachandran** [comment]
> Which one should I use?

@LitoMore

**Comment by LitoMore** [comment]
Hmmm. It seems that we don't need to be too entangled. This difference is not a big deal.

**Review by LitoMore — APPROVED** [review]

**Pull request merged**